src/components/PDFThumbnail/types.ts

```diff
@@ -10,8 +10,8 @@ type PDFThumbnailProps = {
     /** Whether the image requires an authToken */
     isAuthTokenRequired?: boolean;
 
-    /** Callback to call if PDF is password protected */
-    onPassword?: () => void;
+    /** Whether need to skip loading password protected PDF */
+    skipLoadingProtectedPDF?: (() => void) | boolean;
```

Comment:
This prop is acting as two props. Let's avoid that and use two props instead

Reply:
Oh yes, using two props is better. Updated.